web_monitoring/db.py

```diff
@@ -706,6 +710,23 @@ def get_annotation(self, *, annotation_id, page_id, to_version_id,
         data['updated_at'] = parse_timestamp(data['updated_at'])
         return result
 
+    ### USERS ###
+
+    def get_user_session(self):
+        """
+        Get the current user session
+
+        Parameters
+        ----------
+        none
+
+        Returns
+        -------
+        response : dict
+        """
+        user_session_url = f'{self._api_url[:-7]}/users/session'
```

Comment:
I think it’d probably be better to save the `url` that was passed into the constructor separately from `_api_url` and use that here, rather than hacking off an magic number of characters.

Reply:
Good call, removed the magic